src/thunderbots/software/ai/main.cpp

```diff
@@ -40,8 +42,22 @@ void worldUpdateCallback(const thunderbots_msgs::World::ConstPtr &msg)
     }
     primitive_publisher.publish(primitive_array_message);
 
+    // On every tick, send test sprites
+    for (int i = 0; i < 10000; i++)
+    {
+        Util::CanvasMessenger::Sprite sprite;
+        sprite.x        = (i % 100) * 12;
+        sprite.y        = (i / 100) * 12;
+        sprite.width    = 10;
+        sprite.height   = 10;
+        sprite.rotation = count;
+        Util::CanvasMessenger::getInstance()->drawSprite(0, sprite);
+    }
+
     // On every tick, send the layer messages
-    Util::VisualizerMessenger::getInstance()->publishAndClearLayers();
+    Util::CanvasMessenger::getInstance()->publishAndClearLayers();
+
+    count++;
```

Comment:
Not sure if this was left in as a demo, but fair enough if so. Once this has had a round of review we should remove most of the changes to this file and make the `drawSprite` method private so that user code doesn't accidentally use it (this makes things much less painful when we do the next step of abstracting to calls like `drawLine`, `drawRect`, etc.)

Reply:
Removed demo code